### DISCUSSION TITLE
Lesson 6: Brownie Fund Me ValueError: insufficient funds for gas * price + value

### DISCUSSION THREAD

**Discussion by mvarenitsyn** [smartcontractkit/full-blockchain-solidity-course-py#1482] [Q&A]
Hello everyone! 

I'm trying to run a test script (test_fund_me.py) after establishing mainnet-fork-dev using Alchemy. Here is output:

def test_can_fund_and_withdraw():
        account = get_account()
        fund_me = deploy_Fundme()
        print(f"my balance is {account.balance()}")
        entrance_fee = fundme.getEntranceFee() + 100
>       tx = fund_me.fundme({"from": account, "value": entrance_fee})
E       ValueError: insufficient funds for gas * price + value

tests/test_fund_me.py:12: ValueError

What can be a reason? I also check for account balance:
Captured stdout call ----------------------------------------------------------------------------
Contract deployed to 0xE7eD6747FaC5360f88a2EFC03E00d25789F69291
my balance is 1000000000000000000000

Any thoughts?

UPD:
Same code passed well on ganache blockchain.
But on mainnet-fork-dev fund_me.getEntranceFee() function gives value = 238371441489773650809337480

**Comment by cromewar** [comment]
Hello @mvarenitsyn 
Go to ganache settings add setup a higher gas limit,
also be sure to have correct the `getEntranceFee()` function.
